build/ci/delete_unused_disks.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import os
+import json
+
+project = os.environ['GCLOUD_PROJECT']
+
+os.system('gcloud compute disks list --filter="-users:*" --format="json" --project {} > unused_disks.json'
```

Comment:
Does this filter out disks that are currently being used?

Reply:
Yes, it's filtering out all disks which have object `users`. This object contains reference to an instance which uses disk. If disk is not used, then there will be no `users`